js/index.js

```diff
@@ -0,0 +1,37 @@
+
+const replaceSVGTemplatePlaceholders = (template, data) => {
```

Comment:
can we create a class and expose only api's that this library is capable

Reply:
Done